pages/how-to/set-up/new-mgmt-cluster.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Setting Up a New MGMT Cluster
+description: Using Plural CLI to Deploy a MGMT Kubernetes Cluster
+---
+
+### Prerequisites
+[Plural CLI](/how-to/set-up/plural-cli)
+
+##### Ensure Cloud Provider CLI Authentication
+Plural uses the _default_ profile when deploying resources  
+
+AWS  
+```sh
+aws sts get-caller-identity
+```  
+AZ
+```sh
+az account show
+```
+GCP
+```sh
+gcloud auth list
+```
+
+
+### Create a New Repo for Plural
+```sh
+git clone git@github.com:pluralsh/plrl-how-to.git
+cd plrl-how-to
+```
+
+### Setup Repo and Deploy Resources
+Ensure your _[app.plural.sh](https://app.plural.sh/profile/me)_ User has `admin` permissions  
+Follow the onscreen prompts to setup the repo and deploy resources  
+* Use the provided Plural DNS Services for the MGMT Cluster
+* When providing a domain name provide the _canonical_ name, e.g. how-to-plrl.onplural.sh
+```sh
+plural login
+plural up
+```
+
+# Troubleshooting
+### Get Kubeconfig for the MGMT Cluster
+AWS 
+```sh
+aws eks update-kubeconfig --name [CLUSTER_NAME] 
+```
+AZ
+```sh
+az aks get-credentials --name [CLUSTER_NAME]
+```
+GCP
+```sh
+gcloud container clusters get-credentials [CLUSTER_NAME]
+```
+
+Use `kubectl` with the newly added kube context  
+The key namespaces to check are:   
+* plrl-console
+* plrl-deploy-operator
+* plrl-runtime
+
+### "Console failed to become ready"
+Sometimes the DNS Resolution can take longer than the expected five minutes  
+It's also possible the console services take a bit longer to become ready  
+```sh
+2024/07/29 12:31:03 Console failed to become ready after 5 minutes, you might want to inspect the resources in the plrl-console namespace
+```
+In this instance the images in the _`plrl-console`_ namespace  
+were taking a bit longer to download and initialize.  
+Once the services were _up_ in the cli, I was able to access the console url
+
+### Cannot list resources in the Kubernetes Dashboard
+This is due to missing [RBAC Bindings](https://github.com/pluralsh/documentation/blob/8e205adfede17b0e412a2c8d81ac511dd71fe59b/pages/deployments/dashboard.md) for the console users  
+![alt text](/images/how-to/k8s-dash-403.png)
+
+##### Add the RBAC Helm Values in the MGMT Cluster Repo
+```sh
+./helm-values/ConsoleClusterRoleBindings.yaml
```

Comment:
do *not* put this in the `helm-values` folder, that's git-crypted by default so should only be used for secret-ish things, create a new global service in a separate folder for rbac which contains this.  Also don't camel-case filenames for k8s resources (not standard convention)

Reply:
I reference that Here: https://github.com/pluralsh/documentation/pull/289/files/3356e70ddd2f2e93b0baf37b6b500b45b16bcba6#diff-eb5fe659cd5bbca6c330ae4d4e9afe8380837b105bef0a1b1f8c69d126661e7dR97

should the doc just link to that or reiterate what the RBAC readme says?

Reply:
I think this looks and reads better:
![image](https://github.com/user-attachments/assets/4e671189-a573-4f4d-bda6-00bdd6b9f607)

Reply:
and links to: https://github.com/pluralsh/documentation/blob/8e205adfede17b0e412a2c8d81ac511dd71fe59b/pages/deployments/dashboard.md